stablehlo/tests/interpret_ceil.mlir

```diff
@@ -1,75 +1,35 @@
-// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+// RUN: stablehlo-interpreter --interpret -split-input-file %s
 
-// CHECK-LABEL: Evaluated results of function: ceil_op_test_bf16
-func.func @ceil_op_test_bf16() -> tensor<9xbf16> {
-  %0 = stablehlo.constant dense<[0xFF80, -2.5, 0x8001, -0.0, 0.0, 0x0001, 2.5, 0x7F80, 0x7FC0]>  : tensor<9xbf16>
+func.func @ceil_op_test_bf16() {
+  %0 = stablehlo.constant dense<[0xFF80, -2.5, 0x8001, -0.0, 0.0, 0x0001, 2.5, 0x7F80, 0x7FC0]> : tensor<9xbf16>
```

Comment:
Can you provide a comment for what the hex values represent? It will make reading the tests much easier.
```
// -Inf, -2.5, -Denormal, -0.0, 0.0, +Denormal, 2.5, +Inf, +NaN
```
As for `0x7FC0`, can we use `0x7F81` instead? Both are +NaNs, but `0x7F81` is the smallest representable +NaN while `0x7Fc0` falls somewhere in the middle, and the rest of the special values all follow the same convention. I haven't gone through other test values, but they all seem to default to `0x7F80...` depending on the bit length, which is an incorrect assumption for smallest +NaNs depending on the number of exponent bits.

Reply:
As part of https://github.com/openxla/stablehlo/issues/1002, the plan to to revisit the test files interpret_foo.mlir and make them complaint with the checklist. I believe that revision will trim most of these test inputs to something simpler  (most probably to the example test we have in the specification).  

Having said that we should wait till then.